membership/README.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Apereo Membership
+layout: default
+permalink: membership/
```

Comment:
Not only could we, I suspect we already do. :) That plugin is on by default in GitHub Pages iirc.

Reply:
:thinking: Interesting, so does that mean `permalink` isn't needed for this page?

Reply:
I suspect that if we're doing it elegantly we'll seldom need `permalink`. :)

Reply:
It looks like https://github.com/benbalter/jekyll-readme-index is included in the github pages ruby gem, but still needs to be enabled in `_config.yml`

Reply:
And it's already enabled https://github.com/ChristianMurphy/apereo.foundation/blob/8d8ebbfad6a03aab9459bab1048544296f9c7fae/_config.yml#L36

Reply:
Yup. As I understand it (and I may well be wrong), it's not something we could turn off if we wanted to -- it's a thing GitHub Pages does, and then the point of putting it in `_config.yml` is so that non-GitHub-pages deployments where we're developing and testing ([that Docker thing](https://hub.docker.com/r/starefossen/github-pages/) is awesome) will also exhibit the behavior.